components/top-nav/src/index.tsx

```diff
@@ -84,7 +84,7 @@ export class TopNav extends Component<TopNavProps, { navigationOpen: boolean }>
 
   static displayName = 'TopNav';
 
-  static defaultProps = {
+  static defaultProps: TopNavProps = {
```

Comment:
thanks for the PR!

Any reason this was added in? I'm not sure defaultProps should be typed as TopNavProps.

If a required prop was added in the future, it wouldn't be required in the default props.

I think this is fine untyped, as it was.

Reply:
thanks for the feedback @penx I have reverted this now